aspnetcore/signalr/configuration.md

```diff
@@ -71,6 +71,7 @@ The following table describes options for configuring SignalR hubs:
 | `StreamBufferCapacity` | `10` | The maximum number of items that can be buffered for client upload streams. If this limit is reached, the processing of invocations is blocked until the server processes stream items.|
 | `MaximumReceiveMessageSize` | 32 KB | Maximum size of a single incoming hub message. |
 | `MaximumParallelInvocationsPerClient` | 1 | The maximum number of hub methods that each client can call in parallel before queueing. |
+| `DisableImplicitFromServicesParameters` | false | Hub method arguments will be resolved from DI if possible. |
```

Comment:
👍

Reply:
How should we handle this? It's a new option in 7.0 but will show up in 6.0 docs since we're trying to stay away from moniker hell.

Reply:
The whole topic should be versioned. If not the whole topic, then we're forced to version the **whole table**. Can do individual rows of a table.

Reply:
We can't version just a topic, the whole page has to be duplicated and versioned

Reply:
That's what I meant ... the content for `aspnetcore-7.0` ..... or just the whole table here. We still go that way when there are only a few version flips.